### PR TITLE
Update librariejs

### PR DESCRIPTION
**Pull request by alvpickmans**
### Purpose

Updating Library extension `librarie.min.js` according to developed on PR [librarie.js - #166](https://github.com/DynamoDS/librarie.js/pull/166)

dynamo-library

### Declarations

Check these if you believe they are true

- [x] The codebase is in a better state after this PR
- [ ] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [ ] The level of testing this PR includes is appropriate
- [ ] User facing strings, if any, are extracted into `*.resx` files
- [ ] All tests pass using the self-service CI.
- [x] Snapshot of UI changes, if any.
- [ ] Changes to the API follow [Semantic Versioning](https://github.com/DynamoDS/Dynamo/wiki/Dynamo-Versions) and are documented in the [API Changes](https://github.com/DynamoDS/Dynamo/wiki/API-Changes) document.
- [ ] This PR modifies some build requirements and the readme is updated

### Release Notes

- Update Dynamo Library with latest style design

### Reviewers
@QilongTang 

### FYIs
@SHKnudsen

### CODE REVIEW

File: src/AssemblySharedInfoGenerator/AssemblySharedInfo.cs [hunk 1]
```diff
@@ -45,7 +45,7 @@
 // to distinguish one build from another. AssemblyFileVersion is specified
 // in AssemblyVersionInfo.cs so that it can be easily incremented by the
 // automated build process.
-[assembly: AssemblyVersion("2.13.0.2733")]
```
Comment (reddyashish) [inline src/AssemblySharedInfoGenerator/AssemblySharedInfo.cs:48]: Please revert the changes to this file.

**Comment by reddyashish** [comment]
Once the conflict is resolved, Looks good to me

File: src/AssemblySharedInfoGenerator/AssemblySharedInfo.cs (hunk 1 above)
Comment (QilongTang) [reply src/AssemblySharedInfoGenerator/AssemblySharedInfo.cs]: Good point! @reddyashish since I happen to be reading this, addressed and made this PR ready for review

**Comment by QilongTang** [comment]
@reddyashish @aparajit-pratap @pinzart @gregmarr Any idea how to address the issue from security scan, there is a neutral check but I could not seem to find the issue details as to which sensitive term it refers to?

**Comment by gregmarr** [comment]
@QilongTang It's in `librarie.min.js` so I would say 99.999% false positive.  If not, then it would be addressed in the `librarie.js` repo.

**Comment by QilongTang** [comment]
> @QilongTang It's in `librarie.min.js` so I would say 99.999% false positive. If not, then it would be addressed in the `librarie.js` repo.

Thanks, that's my thought as well because the scan on the other repo did not report such issue and we use no secrets in librarie.js

**Review by QilongTang — APPROVED** [review]

**Comment by QilongTang** [comment]
hi @reddyashish if you feel proper, approve and merge this since you are the original reviewer 😉

**Review by reddyashish — APPROVED** [review]
